src/connection_string.ts

```diff
@@ -569,7 +568,9 @@ export const OPTIONS = {
           if (['none', 'snappy', 'zlib'].includes(String(c))) {
```

Comment:
related to my other comment, this should use the enum instead of this hardcoded list, lets take the the opportunity to make this a bit better